rfcs/text/0030/group.yml

```diff
@@ -19,18 +19,8 @@
   reusable:
     top_level: true # top level group relates to top level process
     expected:
-      - at: process.entry_leader
```

Comment:
Because of how the tooling works (an issue of ordering, nestings not being created in the right sequence), we have to simplify how we nest user and group... The fact that we won't populate process.user / process.group (in endpoint) because it's already set at the top level will need to be expressed as an implementation detail elsewhere.

Reply:
It is still valuable to capture the *users* (real, effective, saved) and groups for each process since they may differ from the 'main' process.  Ie it's valuable for the process, process.entry_leader process.session_leader, process.group_leader...

Reply:
The way it works now, the user.real, user.effective and user.saved re-uses will not come through to process, process.entry_leader process.session_leader, process.group_leader re-uses, only user itself will come through. 

If this is an issue, we can definitely chat and work through it.

Reply:
update commiitted to remove self nesting.